### DISCUSSION TITLE
d͡z U+0361 on “d” is too height

### DISCUSSION THREAD

**Discussion by kanmurimori** [psb1558/Junicode-font#108] [Show and tell]
![image](https://user-images.githubusercontent.com/20810895/155887714-59ac5e33-5a39-4d79-b586-e898c4892cc3.png)

**Comment by psb1558** [comment]
Thanks for pointing this out. It's a thing I overlooked when I changed the UPM a while ago. I'm going to create an issue from this and will close it when when I post a build with the fix.
